tests/test_marker_sizes.py

```diff
@@ -0,0 +1,46 @@
+"""Tests for marker sizing."""
+
+import numpy as np
+
+from sb_vision import Token
+
+# Homography matrix of a real image.
+homography_from_camera = np.array([
+    [7.08277177e-01, -6.30171882e-03, 9.36194251e-01],
+    [1.08272961e-02, 7.11294182e-01, 2.64415168e+00],
+    [2.14327973e-05, 7.37441645e-06, 7.25640594e-03],
+])
+distance_model = 'c270'
+image_resolution = (1280, 720)
+
+
+def test_marker_size():
```

Comment:
Thanks for adding this, though this test is rather complex to read and doesn't really validate the underlying approach being taken -- it's more just duplicating the logic somewhere else!

I've added some more holistic tests which also validate the underlying approach in 1274f6a8be8210bcf1e5b1f2ae5db4e2a4a8cf96.